docs/book/src/component-config-tutorial/testdata/project/go.mod

```diff
@@ -5,7 +5,7 @@ go 1.20
 require (
 	k8s.io/apimachinery v0.28.1
 	k8s.io/client-go v0.28.1
-	sigs.k8s.io/controller-runtime v0.16.1
+	sigs.k8s.io/controller-runtime v0.16.2
```

Comment:
You need to revert the changes in the code if the goal is just the doc.

Reply:
Yeah, the goal is just the doc. 
Removed, and thanks.